manifests/init.pp

```diff
@@ -64,6 +68,9 @@
   $enable_ostree = $katello::params::enable_ostree,
   $candlepin_event_queue = $katello_devel::params::candlepin_event_queue,
 
+  $pulp_db_username = $katello_devel::params::pulp_db_username,
+  $pulp_db_password = $katello_devel::params::pulp_db_password,
```

Comment:
Which in turn inherits from ```katello::params``` (https://github.com/Katello/puppet-katello_devel/blob/4d38cf0cad1515bf29fd9b5e0047094404082066/manifests/params.pp#L2). Ugly, but it works ;)